### PR TITLE
Jquery docs minor improvements

### PR DESCRIPTION
**Pull request by pablofiumara**
issue #75

### CODE REVIEW

**Comment by pablofiu** [comment]
@scottgonzalez Is there anything else I can help with? Thank you very much in advance.

File: pages/style-guide/prose.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+### Code Examples
+
+- Use examples to illustrate important concepts.
+- Examples should indicate what the expected result will be in comments before code is presented.
+- Break long examples up into shorter sections to aid comprehension.
+- Favor "Right Way" examples over "Wrong Way" examples – there is more than one wrong way to do something, after all.
+- Use good comments so that explanation within prose isn't necessary.
+- Test your code examples before submitting.
+- Follow all [style guides](http://contribute.jquery.org/style-guide/) for your code examples.
+
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: An empty line in here isn't really needed.

File: pages/style-guide/prose.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	- [htmldog.com](http://www.htmldog.com/)
+
+### Code Examples
+
+- Use examples to illustrate important concepts.
+- Examples should indicate what the expected result will be in comments before code is presented.
+- Break long examples up into shorter sections to aid comprehension.
+- Favor "Right Way" examples over "Wrong Way" examples – there is more than one wrong way to do something, after all.
+- Use good comments so that explanation within prose isn't necessary.
+- Test your code examples before submitting.
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: Worth adding is that code samples should match code style guides too.

File: pages/style-guide/prose.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+#### Linking & Referencing Content
+
+- Link to relevant content within the learn.jquery.com site to refer readers to previously covered topics or concepts.
+- Link to the jQuery blog or API documentation when necessary.
+- Use inline hyperlinks to reference relevant content.
+- Acceptable external resources:
+	- [Mozilla Developer Network](https://developer.mozilla.org/)
+	- [Webplatform.org](http://www.webplatform.org/)
+	- [htmldog.com](http://www.htmldog.com/)
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: I would like to see [HTML5 doctor](http://html5doctor.com/) in here too.

File: pages/style-guide/prose.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- Do use humor strategically. When in doubt, err on the side of formality.
+- Do use hyperlinks to refer readers to concepts or topics that have been covered in other sections.
+- Do attribute others.
+- Don't assume the reader will have prior knowledge of topics or concepts, especially when targeting beginner or intermediate audiences.
+- Don't use rhetorical questions.
+- Don't write in first or third person.
+
+#### Linking & Referencing Content
+
+- Link to relevant content within the learn.jquery.com site to refer readers to previously covered topics or concepts.
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: This sentence is kinda learn site specific.

File: pages/style-guide/prose.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+```
+---
+title:  <article title>
+level:  [beginner|intermediate|advance]
+source:  <url of source of the material derived>
+attribution:
+  - Ralph Whitbeck <ralph@email.com>
+  - John Paul <john@email.com>
+---
+```
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: I don't think there are sites still using YAML?

File: pages/style-guide/prose.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: where below?

File: pages/style-guide/prose.md (hunk 5 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: also this shouldn't be learn site specific.

Comment (scottgonzalez) [inline pages/style-guide/prose.md]: Correct, there are no sites using YAML.

File: pages/style-guide/prose.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: We use tabs, this is 4 spaces.

File: pages/style-guide/prose.md (hunk 4 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: Wrap url in backticks

**Comment by arthurvr** [comment]
Doesn't the front-matter need to be in `Contributing to web sites`? (It's already partly covered in there)

File: pages/style-guide/prose.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+### Code Blocks
+
+Code blocks should be set off with triple backticks and should **not** be indented. That is to say, the site prefers the use of "fenced blocks" notation supported by [gfm](http://github.github.com/github-flavored-markdown/).
+
+## Writing Style
+
+Content should be educational and accessible to a broad audience of developers. The primary target audience is beginning to intermediate jQuery users, with advanced jQuery users as a secondary audience. When creating content, please keep one of these audiences in mind as well as the following style guidelines:
+
+### Prose & Grammar
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: `Prose` is a subheading in an article regarding prose?

File: pages/style-guide/prose.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: some sites use XML, not markdown

File: pages/style-guide/prose.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: Enable the right syntax highlighting for this code block. (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)

File: pages/style-guide/prose.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+source:  <url of source of the material derived>
+attribution:
+  - Ralph Whitbeck <ralph@email.com>
+  - John Paul <john@email.com>
+---
+```
+
+### Code Blocks
+
+Code blocks should be set off with triple backticks and should **not** be indented. That is to say, the site prefers the use of "fenced blocks" notation supported by [gfm](http://github.github.com/github-flavored-markdown/).
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: `gfm` does not seem like a good abbreviation for this. People might not know where we're talking about.

Comment (arthurvr) [inline pages/style-guide/prose.md]: > Spell out abbreviated words on first reference, followed by the abbreviation in parentheses. Use abbreviations on second reference.

File: pages/style-guide/prose.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+## Writing Style
+
+Content should be educational and accessible to a broad audience of developers. The primary target audience is beginning to intermediate jQuery users, with advanced jQuery users as a secondary audience. When creating content, please keep one of these audiences in mind as well as the following style guidelines:
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: Is this true for all sites? I mean we shouldn't write specifically targeting beginners on the API docs.

Comment (arthurvr) [inline pages/style-guide/prose.md]: Something like this maybe?

> Keep in mind that you might be writing for experienced developers as well as beginners. This depends on the kind of content you're writing and the site you're writing for.

File: pages/style-guide/prose.md (hunk 5 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @scottgonzalez 
@arthurvr 

Thank you both for your comments. I fail to understand which changes should be made taking into account that there are no sites using YAML and that this should not be learn site specific. Could you explain it to me, please? Thanks in advance.

File: pages/style-guide/prose.md (hunk 6 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: I guess "below" refers to article headers. Could anyone clarify this, please?

File: pages/style-guide/prose.md (hunk 4 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr Thanks for your comment. Which url are you referring to? learn.jquery.com? Or are you saying that we should add another bullet point before title "Linking & Referencing Content"?

File: pages/style-guide/prose.md (hunk 5 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: To be honest, If you don't know how the architecture of our sites is setup, you're probably not the right guy to handle this.

----

The thing is that, in the past, we used YAML for front matters in some sites and now we use JSON, but the example in here still uses YAML.

Also, we mean some of the properties you use in your example are specific for the learn site  (`learn.jquery.com`). Those properties are not needed in other sites.

Remember that the original article was specific for `learn.jquery.com` while this must be true for every site.

File: pages/style-guide/prose.md (hunk 9 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr Excuse me, what do you mean exactly?

File: pages/style-guide/prose.md (hunk 6 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: > I guess "below" refers to article headers. Could anyone clarify this, please?

This did refer to a section that was still in for the learn.jquery.com site, which is removed here.

File: pages/style-guide/prose.md (hunk 10 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr Which part should be applied syntax highlighting to? The first four lines or from line 14 to line 22 (article header metadata)?

File: pages/style-guide/prose.md (hunk 9 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: Most sites use markdown, while the API site uses XML. Once again, if you don't know this kind of things, you probably aren't the right guy to write this up.

File: pages/style-guide/prose.md (hunk 10 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: Yes indeed.

File: pages/style-guide/prose.md (hunk 12 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr 
@scottgonzalez 
Let's hear Scott's opinion over this. Thank you Arthur for pointing this out.

File: pages/style-guide/prose.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
+<script>{
+  "title": "Prose Style Guide"
```
Comment (arthurvr) [inline pages/style-guide/prose.md]: 2 spaces now? We use tabs.

File: pages/style-guide/prose.md (hunk 4 above)
Comment (arthurvr) [inline pages/style-guide/prose.md]: learn.jquery.com

File: pages/style-guide/prose.md (hunk 5 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr Thanks for the clarification. 
You are right: These is my first contribution to this repository so I am not the right guy to handle this. Let's wait to hear other people's opinion.

File: pages/style-guide/prose.md (hunk 9 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: @arthurvr  Thanks for the clarification.

You are right: These is my first contribution to this repository so I am not the right guy to handle this. Let's wait to hear other people's opinion.

File: pages/style-guide/prose.md (hunk 10 above)
Comment (pablofiu) [inline pages/style-guide/prose.md]: So syntax highlighting should be applied to the first four lines? Please, confirm this.

**Comment by agcolom** [comment]
I can confirm the CLA was signed. 
@pablofiu Are you still able to amend the spaces to tabs? Otherwise, I suggest we land this PR and change spaces to tabs ourselves. @arthurvr thoughts?

**Comment by pablofiu** [comment]
@agcolom : I am able to amend the spaces to tabs but do not know which spaces you are referring to. Could you tell me about them, please? Thanks in advance.

**Comment by pablofiu** [comment]
What does CLA mean?

**Comment by dmethvin** [comment]
Is it okay to land this? I was going to make some edits to the pages here but it would be easier to land this first so it doesn't conflict. I can fix the spaces when I land it.

**Pull request closed without merging**